static/src/js/util/accessMethods/__tests__/buildAccessMethods.test.js

```diff
@@ -621,79 +778,57 @@ describe('buildAccessMethods', () => {
-      }
-    })
-  })
+    buildAccessMethods(collectionMetadata, isOpenSearch)
 
-  describe('when swodlr is disabled', () => {
-    test('no swodlr access method is returned', () => {
-      jest.spyOn(getApplicationConfig, 'getApplicationConfig').mockImplementation(() => ({
-        disableSwodlr: 'true'
-      }))
+    expect(buildSwodlrMock).toBeCalledTimes(1)
 
-      const collectionMetadata = {
-        services: {
-          items: [
-            {
-              conceptId: 'S100000-EDSC',
-              longName: 'Mock PODAAC SWOT On-Demand Level 2 Raster Generation (SWODLR)',
-              name: 'Mock PODAAC_SWODLR',
-              type: 'SWODLR',
-              url: {
-                description: 'Service top-level URL',
-                urlValue: 'https://swodlr.podaac.earthdatacloud.nasa.gov'
-              },
-              serviceOptions: {
-                supportedOutputProjections: [
-                  {
-                    projectionName: 'Universal Transverse Mercator'
-                  },
-                  {
-                    projectionName: 'WGS84 - World Geodetic System 1984'
-                  }
-                ]
-              },
-              supportedOutputProjections: [
-                {
-                  projectionName: 'Universal Transverse Mercator'
-                },
-                {
-                  projectionName: 'WGS84 - World Geodetic System 1984'
-                }
-              ],
-              supportedReformattings: null,
-              supportedInputProjections: null,
-              orderOptions: {
-                items: []
-              },
-              variables: {
-                items: []
-              }
-            }
-          ]
+    expect(buildSwodlrMock).toHaveBeenCalledWith({
+      conceptId: 'S100000-EDSC',
+      longName: 'Mock PODAAC SWOT On-Demand Level 2 Raster Generation (SWODLR)',
+      name: 'Mock PODAAC_SWODLR',
+      orderOptions: {
+        items: []
+      },
+      serviceOptions: {
+        supportedOutputProjections: [
+          {
+            projectionName: 'Universal Transverse Mercator'
+          },
+          {
+            projectionName: 'WGS84 - World Geodetic System 1984'
+          }
+        ]
+      },
+      supportedInputProjections: null,
+      supportedOutputProjections: [
+        {
+          projectionName: 'Universal Transverse Mercator'
+        },
+        {
+          projectionName: 'WGS84 - World Geodetic System 1984'
         }
+      ],
+      supportedReformattings: null,
+      type: 'SWODLR',
+      url: {
+        description: 'Service top-level URL',
+        urlValue: 'https://swodlr.podaac.earthdatacloud.nasa.gov'
+      },
+      variables: {
+        items: []
       }
-      const isOpenSearch = false
+    })
+  })
 
-      const methods = buildAccessMethods(collectionMetadata, isOpenSearch)
+  describe('calls complex compilation of mutliple different access methods', () => {
```

Comment:
Are the next three tests supposed to be nested within this describe block, or is this describe block a placeholder for test that wasn't written yet?

Reply:
They are supposed to be nested in this describe block

Reply:
Those tests should be moved inside the describe block.

Reply:
Adding another comment just so I can mark this as a review.

Reply:
Seems like I don't need to add this test. I'm just going to remove it.